lib/seedQuestions/index.js

```diff
@@ -109,6 +110,23 @@ class QuestionSeeder {
             });
         }
 
+        // updating config file once you run the script.
+        if (!this.updateConfigFile){
+            console.log("Updating Config file.");
+            await this.UpdateToConfig(this.config);
+        }
+    }
+
+
+    async UpdateToConfig(config) {
+        // update config file if new buckets choice is added.
+        _.each(config.buckets, (updateConfig, index) => {
+            _.each(updateConfig.choices, (choice, indx) => {
+                config.buckets[index].choices[indx].file = choice.file.substr(10)
```

Comment:
the file paths you have in the config variable should be fine to write as it is as they were paths before also. so why is this being done?

Reply:
I use *substr* because while updating config.json file this.config.file return full path of question dir if user want's to add questions against then it's throwing the error question file does not exist. that's why I have ignore the question string using substr.